### ISSUE TITLE
network/web interface

### ISSUE THREAD

**Issue by nick87720z** [KillerInk/FreeDcam#105]
There could be some limited web interface, to control phone's FreeDCam via network. As I mentioned in #103, live preview is not critical - in-progress downloading/viewing files at PC looks more reliable, at least when it comes to using big expo times with low to min sensitivity (since I never have seen camera apps using too big expo for live preview).

**Comment by KillerInk** [comment]
yeah nanohttp can do it. it should be not so hard to setup a page with parameters attached.
